extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -379,6 +382,10 @@ class AmpAd extends AMP.BaseElement {
    * @private
    */
   noContentHandler_() {
+    // If iframe is null nothing to do.
```

Comment:
This still leaves us open to the deferred issue. If the ad calls `#noContentHandler_` then the ad is unloaded, it will try to access the iframe's name and the iframe is null.